### ISSUE TITLE
Ensure training pipeline has reliable validation sets

### ISSUE THREAD

**Issue by jackapbutler** [OpenBioML/chemnlp#262]
This is currently concerned with preventing the dataset shuffling when the pipeline calls `Dataset.train_test_split()` by setting `shuffle=False`. This makes the validation loss reliable across grid search runs as we can be sure each run is using the same split.
